apps/docs/src/app/core/component-docs/dialog/examples/dialog-backdrop-container/dialog-backdrop-container-example.component.ts

```diff
@@ -1,10 +1,10 @@
-import { Component, ViewEncapsulation } from '@angular/core';
+import { Component } from '@angular/core';
 import { DialogService } from '@fundamental-ngx/core/dialog';
 
 @Component({
     selector: 'fd-dialog-backdrop-container-example',
     templateUrl: './dialog-backdrop-container-example.component.html',
-    encapsulation: ViewEncapsulation.None,
+    styleUrls: ['../dialog-examples.component.scss'],
```

Comment:
Please also include this file into stackblitz example. You can check how it's done in object status examples.